website/styles/globals.css

```diff
@@ -0,0 +1,55 @@
+html {
+  font-family: 'Akzidenz', Helvetica, Arial, sans-serif;
+}
+
+body {
+  margin: 0;
+}
+
+*,
+*:before,
+*:after {
+  box-sizing: border-box;
+}
+
+@font-face {
+  font-family: 'Akzidenz';
+  font-weight: normal;
+  src: url('fonts/akzidgrostdreg.eot');
+  /* IE9 Compat Modes */
+  src: url('fonts/akzidgrostdreg.eot?#iefix') format('embedded-opentype'),
+    /* IE6-IE8 */ url('fonts/akzidgrostdreg.woff') format('woff'),
+    /* Pretty Modern Browsers */ url('fonts/akzidgrostdreg.ttf')
+      format('truetype'),
+    /* Safari, Android, iOS */ url('fonts/akzidgrostdreg.svg#Akzidenz')
+      format('svg');
+  /* Legacy iOS */
```

Comment:
hm, nothing for legacy ios?

Reply:
Ah grabbed this from our Storybook styles, cc: @DesignerDave

Reply:
I can never remember every part of resilient font-face declarations, so I probably copy + pasted this hah

Legacy iOS isn't something we need to cover, due to the aggressive adoption of new iOS versions by users, and aggressive backwards-compatibility of iOS versions.